package.json

```diff
@@ -30,7 +30,7 @@
         ]
     },
     "scripts": {
-        "start": "parcel public/index.html",
+        "start": "parcel public/index.html && parcel watch public/index.html",
```

Comment:
This is a bit like duplicating the command. You should just be able to run 'parcel watch public/index.html'